sys/socket_unix.go

```diff
@@ -7,11 +7,16 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/pkg/errors"
 	"golang.org/x/sys/unix"
 )
 
 // CreateUnixSocket creates a unix socket and returns the listener
 func CreateUnixSocket(path string) (net.Listener, error) {
+	// BSDs have a 104 limit
+	if len(path) > 104 {
+		return nil, errors.Errorf("%q: unix socket path too long (> 106)", path)
```

Comment:
just realize I have a typo here, 106 !=  104 :innocent:

Reply:
:man_facepalming:

Reply:
Can you fix that in another PR and I'll backport that as well?

Reply:
sure